api/Window.json

```diff
@@ -2180,10 +2180,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/external",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "1"
```

Comment:
`window.external` is a bit special, I think that for a long time it wasn't defined using Web IDL a bit like `console`. Have you checked what kind of thing it is in Chrome 1, was it an object with methods on it? I'm worried it might be a similar situation to `window.crypto` above.